modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -181,12 +181,9 @@ mount -o loop /tmp/<iso_filename> /srv/www/htdocs/pub/rhel
 
 When the image is mounted, you can synchronize the base media.
 To manually synchronize the channels, navigate to menu:Software[Manage > Channels].
-Click each channel in the list, and:
-
-. Navigate to the [guimenu]``Repositories`` tab.
-. Navigate to the [guimenu]``Sync`` subtab.
-. Click btn:[Sync Now] to begin synchronization immediately.
-. You can also create a synchronization schedule from this screen.
+Click each channel in the list, and navigate to menu:[Repositories > Sync].
+Click btn:[Sync Now] to begin synchronization immediately.
+You can also create a synchronization schedule from this screen.
```

Comment:
```suggestion
* You can also create a synchronization schedule from this screen.
```
Maybe, a simple itemized list would make sense?  If not, ignore my proposal.  It's not that important.

Reply:
It's got to be a numbered list, because the order is important. So we either turn it into prose, or we make it a proper procedure. Making it a procedure seem like overkill to me, but I could be persuaded :wink:

Reply:
Thinking about it again, I'd rather vote to shorten it and turn it into prose.  I hope I edited my previous comment correctly...